integration-test/test_dummy_data.py

```diff
@@ -0,0 +1,185 @@
+# Note that this assumes that the KNMI test data is loader (using loader container)
```

Comment:
Does it?

Reply:
It does not, I used the other file as starting point, simply copied it and did not notice the top comment.